tests/queries/0_stateless/02012_get_port.sql

```diff
@@ -0,0 +1,3 @@
+select getPort('tcp'), getPort('tcp_with_proxy'), getPort('tcp_secure'), getPort('http'), getPort('https'), getPort('interserver_http'), getPort('interserver_https'), getPort('postgresql'), getPort('mysql'), getPort('grpc'), getPort('prometheus');
```

Comment:
I'm afraid that the name `getPort` is not specific enough to indicate that it is just config introspection function.

Reply:
How about `getServerPort` or `getConfigPort` or `getServerListeningPort` ?

Reply:
getServerPort looks ok.